app/posts/manage-teacher-training-applications/2021-03-22-application-fee-status-guidance.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Show guidance on a candidate’s tuition fee status
+description: Let users see when a candidate’s application is eligible for home tuition fee funding.
+date: 2021-03-22
+---
+
+It’s possible to work out from a candidate’s application whether or not they’re likely to qualify for ‘home tuition fee funding’. This has implications for how the provider’s course is funded as well as for the candidate’s ability to get things like student loans.
```

Comment:
It's possible for who to work out?

What is ‘home tuition fee funding’?

Funded in what way?

What other things besides student loans?